vite.config.mts

```diff
@@ -1,10 +1,14 @@
-import path from 'path';
+/// <reference types="vitest" />
 import { defineConfig } from 'vite';
+import path from 'path';
 import vue from '@vitejs/plugin-vue';
 import svg from 'vite-svg-loader';
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  test: {
+    environment: 'jsdom',
+  },
```

Comment:
It works even without using `import { defineConfig } from 'vitest'`?

Reply:
As I understood, newer versions of vite support this field. Additionally, it even works without using reference line
```
/// <reference types="vitest" />
import { defineConfig } from 'vite';
```
But I would keep it.

Reply:
Cool! Good to know.